axelor-supplychain/src/main/resources/views/MrpForecast.xml

```diff
@@ -76,12 +76,24 @@
     <field name="technicalOrigin" expr="eval: 1"/>
   </action-record>
 
-  <action-attrs name="action-mrp-forecast-attrs-confirm-btn-on-click">
-    <attribute name="value" for="statusSelect" expr="eval: 2"/>
-  </action-attrs>
+  <action-method name="action-mrp-forecast-method-confirm">
+    <call class="com.axelor.apps.production.web.MrpForecastController" method="confirm"/>
+  </action-method>
 
-  <action-attrs name="action-mrp-forecast-attrs-cancel-btn-on-click">
-    <attribute name="value" for="statusSelect" expr="eval: 3"/>
-  </action-attrs>
+  <action-method name="action-mrp-forecast-method-cancel">
+    <call class="com.axelor.apps.production.web.MrpForecastController" method="cancel"/>
+  </action-method>
+
+  <action-group name="action-mrp-forecast-group-confirm-button">
+    <action name="save"/>
+    <action name="action-mrp-forecast-method-confirm"/>
+    <action name="save"/>
```

Comment:
Please remove the last save, now that we use an action method fetching from db it is not necessary anymore.

Reply:
Done